include/flags.hpp

```diff
@@ -711,7 +711,7 @@
 // free                             114
 // free                             115
 // free                             116
-// free                             117
+#define O_SEL_KENKU                 117      // race selective: kenku
```

Comment:
is there nothing closer to the other ones? I recall it being some math to figure out which flag it was.

Reply:
It checks the flags individually. No math.